doc/source/whatsnew/v0.19.0.txt

```diff
@@ -618,6 +618,37 @@ Furthermore:
 - Passing duplicated ``percentiles`` will now raise a ``ValueError``.
 - Bug in ``.describe()`` on a DataFrame with a mixed-dtype column index, which would previously raise a ``TypeError`` (:issue:`13288`)
 
+.. _whatsnew_0190.api.perioddtype:
+
+``PeriodIndex`` now has ``period`` dtype
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``PeriodIndex`` now has its own ``period`` dtype. ``period`` dtype is a
+original dtype which extends NumPy dtype like ``DatetimeTZDtype``
+for datetime with timezone. (:issue:`13941`)
```

Comment:
``DatetimeTZDtype`` is really an internal thing, maybe just like: ``datetime64[ns, US/Eastern]`` or something